src/pybind/mgr/influx/module.py

```diff
@@ -111,7 +111,22 @@ def send_to_influx(self):
         username = self.get_config("username", default="")
         password = self.get_config("password", default="")
 
-        client = InfluxDBClient(host, port, username, password, database)
+        # Use https connection to influx
+        ssl = self.get_config("ssl", default="false")
+        if ssl.lower() == "true":
+            ssl = True
+        else:
+            ssl = False
+
+        # Verify https cert
+        verify_ssl = self.get_config("verify_ssl", default="false")
```

Comment:
Can we make verify_ssl true by default?  It feels a bit dangerous to have a "ssl=true" setting that by default doesn't validate the certs.

Reply:
the influxdb client also defaults to `false`